packages/client/src/app/game/views/game-container/game-container.component.ts

```diff
@@ -23,15 +23,15 @@ import * as _ from 'lodash';
 })
 export class GameContainerComponent implements OnInit, OnDestroy {
   public isViewer: boolean;
-  private gameData$: Observable<GameFields.Fragment>;
+  gameData$: Observable<GameFields.Fragment>;
```

Comment:
ERROR in /home/ubuntu/CesiumFPS/packages/client/src/$$_gendir/app/game/views/game-container/game-container.component.ngfactory.ts (98,31): Property 'otherPlayers$' is private and only accessible within class 'GameContainerComponent'.

  ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-container/game-container.component.html (2,11): Property 'gameData$' is private and only accessible within class 'GameContainerComponent'.

ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-container/game-container.component.html (5,3): Pro  rty 'me' is private and only accessible within class 'GameContainerComponent'.

ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-container/game-container.component.html (7,3): Pro  rty 'gameResult$' is private and only accessible within class 'GameContainerComponent'.